source/plugin/workspace/idea.rst

```diff
@@ -5,11 +5,57 @@ Setting Up IntelliJ IDEA
 This article describes how to configure your **IntelliJ IDEA** workspace for plugin development with SpongeAPI and
 :doc:`a build system such as Maven or Gradle <../buildsystem/>` or the `Minecraft Development plugin <https://minecraftdev.org/>`_.
 
-Gradle
-======
+If you want to create your project completely from scratch, please skip ahead to the Gradle or Maven sections.  
+Using the Minecraft Dev plugin sets up a working starting point and eliminates some of the guesswork in getting
+your project off the ground.
 
-Creating Your Project
-~~~~~~~~~~~~~~~~~~~~~
+Using IDEA Minecraft Dev Plugin to Create a Working Starting Point
+==================================================================
+
+The `Minecraft Development plugin <https://minecraftdev.org/>`_ for IntelliJ is a great plugin by a community
+member which makes plugin project creation much easier while also providing some neat and useful features for
+development.  By default it will create a project which uses Gradle as the build tool.
+
+Installing the Minecraft Dev Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin is available on the JetBrains IntelliJ plugin repository.
+
+Because of this, you can install the plugin through IntelliJ's internal plugin browser. Navigate to
+``File -> Settings -> Plugins`` and click the ``Browse Repositories...`` button at the bottom of the
+window. In the search box, simply search for ``Minecraft``. You can install it from there and
+restart IntelliJ to activate the plugin.
+
+Creating Your Project from a Template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Open **IntelliJ IDEA**.
+* Click ``Create New Project``.
+* Select ``Minecraft`` in the popup.
+* Make sure your **Project SDK** is set to some version of Java 8/1.8.
+* Select ``Sponge plugin`` for your project type, then click ``Next``.
+* Enter your **Group ID**, **Artifact ID**, and **Version**.
+
+  * Your **Group ID** should usually correspond to your Java package name. See :doc:`../plugin-class` for details.
+  * Your **Artifact ID** should usually correspond to your **plugin ID** you chose earlier, e.g. ``myplugin``.
+  * Your **Version** is up to you.
+
+* Select your desired build tool, either Gradle or Maven, and click ``Next``.
+* Check your **Plugin Name** and **Main Class Name** to make sure they are what you want.
+* Specify your desired plugin **description**, **authors**, **website**, and **dependencies** if you want.
+* Click ``Next`` to move on.
+* Verify your project name, location, and module information, then click ``Finish``.
+* The plugin will create a main java file as a starting point, with the logger already injected.
+  You may add a ``logger.info`` statement in the ``onServerStart`` event handler to verify that the plugin
```

Comment:
Maybe add some brackets to `logger.info()` to make it clearer that this refers to a method call.